table/block_based/block_based_table_builder.cc

```diff
@@ -994,33 +995,39 @@ void BlockBasedTableBuilder::Flush() {
                                              r->get_offset());
     r->pc_rep->EmitBlock(block_rep);
   } else {
-    WriteBlock(&r->data_block, &r->pending_handle, true /* is_data_block */);
+    CachableEntry<Block> entry;
```

Comment:
We don't necessarily need to propagate this all the way down to ```InsertBlockInCache```, since the ```CachableEntry``` will not be used in this path.